streamloader/src/main/resources/events.scala

```diff
@@ -0,0 +1,197 @@
+/*
```

Comment:
The only place where it is actually useful is in `src/main/scala`. It's meant to be used when creating a `StaticSource`, so that you can run the app locally with a mock event stream instead of requiring a PubSub topic of enriched events. It cannot be imported from `src/test` nor from it's current location. I thought adding it to `src/main/resources` will keep it visible while not messing up `src/main/scala`. But it has to be moved manually there whenever we want to actually use it.

I'm not a huge fan of the current setup but I see it as the least of all evils. But very open to ideas how to improve it!